src/include/btmem.h

```diff
@@ -960,15 +946,71 @@ struct __wt_ref {
 #define ref_ikey key.ikey
 
     /*
-     * Fast-truncate information. When a WT_REF is included in a fast-truncate operation, WT_REF.del
-     * is allocated and initialized. If the page must be instantiated before the truncate becomes
-     * globally visible, WT_UPDATE structures are created for the page entries, the transaction
-     * information from WT_REF.del is migrated to those WT_UPDATE structures, and the WT_REF.del
-     * field is freed and replaced by the WT_REF.update array (needed for subsequent transaction
-     * commit/abort). Doing anything other than testing if WT_REF.del/update is non-NULL (which
-     * eviction does), requires the WT_REF be locked. If the locked WT_REF's previous state was
-     * WT_REF_DELETED, WT_REF.del is valid, if the WT_REF's previous state was an in-memory state,
-     * then WT_REF.update is valid.
+     * Fast-truncate information, written-to/read-from disk as necessary in the internal page's
+     * deleted page proxy cell. When a WT_REF first becomes part of a fast-truncate operation, the
+     * ft_info.del field is allocated and initialized.
+     *
+     * Fast-truncate pages might have to be instantiated if a thread for which the operation isn't
+     * visible accesses the page. This can happen if the operation hasn't committed yet; it can also
+     * happen if an older read transaction visits the page; and it can happen if the fast-truncate
```

Comment:
Two semi-colons in a row, maybe change the second to a comma.

Reply:
Fixed in [42d20ae](https://github.com/wiredtiger/wiredtiger/pull/7909/commits/42d20ae234f880c06e2cc2133a5c772b2c7414a6)